extension/content.css

```diff
@@ -196,11 +196,13 @@
 }
 
 /* indicate collapsible file headers with zoom cursor */
-.file-header[data-path], .file-header[data-path] .diffstat {
-	cursor: zoom-out !important;
+.js-details-container:not(.discussion-item) .file-header[data-path],
+.file-header[data-path] .diffstat {
+    cursor: zoom-out !important;
 }
-.refined-github-minimized .file-header[data-path], .refined-github-minimized .file-header[data-path] .diffstat {
-	cursor: zoom-in !important;
+.refined-github-minimized:not(.discussion-item) .file-header[data-path],
+.refined-github-minimized .file-header[data-path] .diffstat {
+    cursor: zoom-in !important;
```

Comment:
I've overlooked the diffstat thing in my previous comment though. I think we would need the `:not()` thing for the `.diffstat` selector as well. What do you think?